cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/local/DataStorageCcpa.kt

```diff
@@ -160,15 +154,6 @@ private class DataStorageCcpaImpl(context: Context) : DataStorageCcpa {
             .apply()
     }
 
-    override var ccpaPostChoiceResp: String?
```

Comment:
did you confirm that we are not using this value at all and we do not need to store it?

Reply:
Yep.